crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -199,6 +230,17 @@ pub struct QuerySetState<T>(T);
 
 impl_query_set!();
 
+pub struct ParamSet<'w, 's, T: SystemParam> {
```

Comment:
This is an important bit of the public API and needs doc strings describing its purpose and a little example demonstrating its use <3

Reply:
done, making descriptive examples is maybe not my forte though (and `QuerySet` had nothing to work from)

Reply:
Example seems perfectly serviceable for API docs :) I'll toss something nicer together for the book once this is merged.